tests/Makefile

```diff
@@ -0,0 +1,30 @@
+#adapted from Lab5
```

Comment:
you should probably remove this, it's not meaningful for people not in this class